<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Drillhole Database (for saving results and resuming)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Drillhole Database</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="DrillholeDatabase" />
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="Resume" >
        <property name="text" >
         <string>Resume Previous Run</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="ResumeFrom" />
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="RefreshButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Genetic Algorithm Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="1" >
       <widget class="QSpinBox" name="NumGenerations" >
        <property name="maximum" >
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="NumPasses" >
        <property name="maximum" >
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="PopulationSize" >
        <property name="suffix" >
         <string/>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="IndividualSize" >
        <property name="suffix" >
         <string> Drillholes</string>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Number of Generations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Number of Hill Climbing Passes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Population Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Individual Size</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="MutationProb" >
        <property name="suffix" >
         <string>%</string>
        </property>
        <property name="maximum" >
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QDoubleSpinBox" name="MutationGeneProb" >
        <property name="suffix" >
         <string>%</string>
        </property>
        <property name="maximum" >
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Probability of Mutation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Probability of Gene Mutation</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Drillhole Segment Length</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QDoubleSpinBox" name="SegmentLength" >
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Hill Climbing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="DoHillClimbLength" >
        <property name="text" >
         <string>On Length of Drillhole</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="HillClimbLength" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="DoHillClimbDirection" >
        <property name="text" >
         <string>On Direction of Drillhole</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="HillClimbAzimuthWindow" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QDoubleSpinBox" name="HillClimbDipWindow" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_22" >
        <property name="text" >
         <string>Azimuth</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_23" >
        <property name="text" >
         <string>Dip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Drillhole Constraints</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="DAzimuthMin" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QDoubleSpinBox" name="DDipMax" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="DAzimuthMax" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="DDipMin" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-90.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Azimuth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Dip</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Minimum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="DLengthMin" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QDoubleSpinBox" name="DLengthMax" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Length</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_5" >
     <property name="title" >
      <string>Anisotropy Values</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" >
      <item row="0" column="1" >
       <widget class="QLabel" name="label_19" >
        <property name="text" >
         <string>Major</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_20" >
        <property name="text" >
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_21" >
        <property name="text" >
         <string>Minor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Ellipsoid Magnitude</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="EMajor" >
        <property name="prefix" >
         <string/>
        </property>
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="EMedium" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="EMinor" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>Azimuth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_17" >
        <property name="text" >
         <string>Dip</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Rake</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Ellipsoid Angles</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QDoubleSpinBox" name="EAzimuth" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QDoubleSpinBox" name="EDip" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QDoubleSpinBox" name="ERake" >
        <property name="suffix" >
         <string> Degrees</string>
        </property>
        <property name="minimum" >
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_6" >
     <property name="title" >
      <string>Drillhole Collar Locations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7" >
      <item row="1" column="0" colspan="2" >
       <widget class="QLineEdit" name="CollarFile" />
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="CollarLocation" >
        <item>
         <property name="text" >
          <string>CSV File</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>In Filter "Source"</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_24" >
        <property name="text" >
         <string>Where are the drillhole collars?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLineEdit" name="ResumeFromC2S" />
   </item>
   <item row="10" column="0" >
    <widget class="QCheckBox" name="RefreshPlease" >
     <property name="text" >
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
